Common/TableProducer/trackselection.cxx

```diff
@@ -53,32 +56,29 @@ struct TrackSelectionTask {
         break;
       case 1:
         // Run 3 kAny on 3 IB layers of ITS
-        if (isRun3) {
-          globalTracks = getGlobalTrackSelectionITSMatch(TrackSelection::GlobalTrackRun3ITSMatching::Run3ITSibAny);
-        }
+        globalTracks = getGlobalTrackSelectionITSMatch(TrackSelection::GlobalTrackRun3ITSMatching::Run3ITSibAny);
```

Comment:
This cut is for sure meaningful only for `Run 3` only analyses. With the idea to keep the `bool isRun3` and following your proposal, I suggest to write what follows:

`case 1:
     globalTracks = getGlobalTrackSelectionITSMatch(TrackSelection::GlobalTrackRun3ITSMatching::Run3ITSibAny);
     isRun3 = true;
     LOG(info) << ... ;`

where the `LOG(info)` is meant to advice that the `isRun3` boolean was just set to `true` because of the chosen ITS matching.

The same suggestion holds also for the other two cases below.